tools/depth-quality/depth-quality-model.cpp

```diff
@@ -63,20 +63,13 @@ namespace rs2
 
             int requested_fps = usb3_device ? 30 : 15;
```

Comment:
`requested_fps` is no longer in use, why is that?
@mengyui , 
Don't we want to limit fps for USB2?
What will be the default resolution for D435? we need to make sure we don't exceed the baud rate.
If all cases are covered, this line can be dropped.
@Tamir91 FYI